proto/rill/admin/v1/api.proto

```diff
@@ -249,6 +249,32 @@ service AdminService {
     option (google.api.http) = {get: "/v1/superuser/resource"};
   }
 
+  // SudoGetUserQuota returns quotas available for users
+  rpc SudoGetUserQuota(SudoGetUserQuotaRequest) returns (SudoGetUserQuotaResponse) {
+    option (google.api.http) = {get: "/v1/superuser/quotas/user"};
+  }
+
+  // SudoSetUserQuota update the quotas for users
+  rpc SudoSetUserQuota(SudoSetUserQuotaRequest) returns (SudoSetUserQuotaResponse) {
+    option (google.api.http) = {
+      post: "/v1/superuser/quotas/user",
+      body: "*"
+    };
+  }
+
+  // SudoGetOrganizationQuota returns quotas available for orgs
+  rpc SudoGetOrganizationQuota(SudoGetOrganizationQuotaRequest) returns (SudoGetOrganizationQuotaResponse) {
+    option (google.api.http) = {get: "/v1/superuser/quotas/organization"};
+  }
```

Comment:
Make getting quotas part of the `GetOrganization` API (can add a `OrganizationQuotas quotas = x;` field on the `Organization` type). Then change the access in that func to allow superuser access to all orgs.